quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/dashboard/domain/DashboardRepository.java

```diff
@@ -19,10 +20,10 @@ public DashboardRepository(WidgetConfigService widgetConfigService) {
         this.widgetConfigService = widgetConfigService;
     }
 
-    public List<Dashboard> findAll() {
+    public Page<Dashboard> findAll(Pageable pageable) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "pageable". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)